src/control/pagetype/PageTypeHandler.cpp

```diff
@@ -117,3 +121,82 @@ vector<PageTypeInfo*>& PageTypeHandler::getPageTypes()
 
 	return this->types;
 }
+
+PageTypeFormat PageTypeHandler::getPageTypeFormatForString(const string& format)
```

Comment:
This is nicer to implement with a static const std::map

Reply:
`static const std::map` is unfortunately not a nice way to do this as it may throw a non-catchable exception at initialization. Additionally for a map of this size the runtime will be equal to running through a few conditions.